Output Teacher Overview + Appearance

mestrona3 opened the pull request:
Alyssa created the output teacher overview and I made some changes to it so that it is readable.

Some minor fixes were also needed to fix errors (i.e. just use cohortNames.size())

To test:
- run model
- open flamborough sheet, navigate to last output (just created)
- check to make sure text is wrapped, top row with day names is bolded and all model results expected (i.e. cohort name and subject appear in matrix), teacher names as column
- make sure that 'prep' and 'away' are listed in each applicable cell, without any slashes or other information

mestrona3 committed:
Output Teacher Overview + Appearance

Alyssa created the output teacher overview and I made some changes to it so that it is readable.

mccurdy42 commented:
I have run it 3x and a solution is found on eclipse, but the output is not creating a new sheet in excel and is not printing to the console.

ajacobs12 reviewed (dismissed):
I was able to get this running on my local computer. That said, I think there are some issues with what is being printed. Classes 8b and 8c are actual cohorts but I think are being looked at as the dummy cohorts for prep and away. In the schedule, they are not showing a subject as assigned to them. Only the cohort name. See picture below: 
![test](https://user-images.githubusercontent.com/30099342/74255286-a2157080-4cbf-11ea-8a36-24ca968f4174.PNG)

The dummy cohorts needed to be added to the cohort names array list in order for the actual subject to be printed. It also looks like the size of n3 was changed to just be cohortnames.size but it needs to have the extra 2 added to account for the additional dummy cohorts. Pic below:
![image](https://user-images.githubusercontent.com/30099342/74255616-1f40e580-4cc0-11ea-915f-745dc7879741.png)

While these changes are being committed, can we also fix the merge conflicts? That way once this is resolved, the PR is able to be merged right away!

ahopkins19 commented on FYDP_ModV1/src/version1.java:
```diff
@@ -268,6 +276,7 @@ public static void modelConfig() throws IOException {
         	
         	if (inputSheet2.getRow(gradeNameStartRow).getCell(gradeNameStartCol+k).getCellType() == CellType.STRING) {
         		gradeNames.add(inputSheet2.getRow(gradeNameStartRow).getCell(gradeNameStartCol+k).getStringCellValue());
+        		cohortNames.add(inputSheet2.getRow(cohortNameStartRow).getCell(gradeNameStartCol+k).getStringCellValue());
```
instead of gradeNameStartCol should be cohortNameStartCol

mestrona3 committed:
Added in prep and away

Output now is shown as expected, with cohorts and subjects for each teacher other than away and prep which are also shown.

ahopkins19 commented on FYDP_ModV1/src/version1.java:
```diff
@@ -267,7 +275,8 @@ public static void modelConfig() throws IOException {
         	
         	if (inputSheet2.getRow(gradeNameStartRow).getCell(gradeNameStartCol+k).getCellType() == CellType.STRING) {
-        		gradeNames.add(inputSheet2.getRow(gradeNameStartRow).getCell(gradeNameStartCol+k).getStringCellValue());
+        		gradeNames.add(inputSheet2.getRow(gradeNameStartRow).getCell(gradeNameStartRow+k).getStringCellValue());
```
gradeNames.add(inputSheet2.getRow(gradeNameStartRow).getCell(gradeNameStart**Row**+k).getStringCellValue());
this should be gradeNameStartCol in the .getCell() part

mestrona3 committed:
GradeNameStartCol

fixed variable

ajacobs12 committed:
clearing excel file

clearing excel file to help with merge errors?

ajacobs12 committed:
capitalizing

capitalizing away and prep

mccurdy42 reviewed (approved):
I have run this branch and it runs in an acceptable range and with the desired output
